CONTRIBUTORS.md

```diff
@@ -7,26 +7,33 @@ Chapel release:
 Contributors to the current release
 -----------------------------------
 * Ben Albrecht, [Cray Inc.]
-* Sidhant Bendre, individual contributor
+* Akshansh Bhanjana, individual contributor
 * Paul Cassella, [Cray Inc.]
 * [Brad Chamberlain], [Cray Inc.]
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology, Roorkee]
+* Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, SriCity] (former individual contributor)
```

Comment:
Can you correct the name of Institute to `Indian Institute of Information Technology, Sri City`.

Reply:
Yes, thanks for catching that.  I took it from their LinkedIn page, thought it seemed odd, and then failed to notice that their actual webpage had the space.

Reply:
Thanks. I need to update on my LinkedIn page as well.